OpenRA.Game/Settings.cs

```diff
@@ -207,6 +207,11 @@ public class KeySettings
 		public Hotkey MapScrollLeft = new Hotkey(Keycode.LEFT, Modifiers.None);
 		public Hotkey MapScrollRight = new Hotkey(Keycode.RIGHT, Modifiers.None);
 
+		public Hotkey MapPushTop = new Hotkey(Keycode.UP, Modifiers.Ctrl);
```

Comment:
These keys are quite inconvenient for OSX because ctrl+left/right are used to switch desktops and ctrl+up is used to enter mission control.  This is roughly equivalent to minimizing the game or alt-tabbing under windows.